ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -263,4 +263,25 @@ - (void)testShouldHandleVnodeOpEvent {
     XCTAssertEqual(kauthResult, KAUTH_RESULT_DEFER);
 }
 
+//KEXT_STATIC void UseMainForkIfNamedStream(vnode_t& vnode, bool& putVnodeWhenDone);
```

Comment:
Minor: We don't have comments like this:

```//KEXT_STATIC void UseMainForkIfNamedStream(vnode_t& vnode, bool& putVnodeWhenDone);```

On the other tests.  I think it can probably be removed.